api/odl.py

```diff
@@ -102,7 +105,13 @@ class ODLWithConsolidatedCopiesAPI(BaseCirculationAPI):
         {
             "key": Collection.DATA_SOURCE_NAME_SETTING,
             "label": _("Data source name"),
-        }
+        },
+        {
+            "key": Collection.DEFAULT_RESERVATION_PERIOD_KEY,
+            "label": _("Default Reservation Period (in Days)"),
+            "type": "number",
+            "default": Collection.STANDARD_DEFAULT_RESERVATION_PERIOD,
+        },
```

Comment:
This needs a `description` to help admins distinguish between a 'reservation period' and the period where you have something on hold and you're waiting to get to the front of the queue.